### DISCUSSION TITLE
Menus fall of display on Plug-In GUI

### DISCUSSION THREAD

**Discussion by sparksandshadows** [apohl79/audiogridder#568] [General]
![image](https://user-images.githubusercontent.com/82538597/118881613-246be780-b8a8-11eb-8263-6ed17e0aa87d.png)

If you look at the image you'll see the drop down menus fall off the side of the screen. Tying back into the plug-in UI virtualization, one needs to basically trick the GUI into thinking there's nowhere to go I think in order for drop down menus to stay in the display. Any thoughts?

**Comment by apohl79** [comment]
You can extend the capture area by using the screen controls at the top right corner of the plugin window.

**Reply by sparksandshadows** [reply]
Got it! Thank you!
